buildSrc/src/main/kotlin/io.customer/android/Configurations.kt

```diff
@@ -1,7 +1,7 @@
 package io.customer.android
 
 object Configurations {
-    const val compileSdk = 30
+    const val compileSdk = 33
```

Comment:
Yes and no, we can downgrade dependencies to be compatible with 30 but since 34 version is out, i want to be at least sure that we are able to compile with SDK version 33. 

Note: This doesn't enforce our customers to adopt this version, it just compiles our SDK with this version to make sure its compatible with it.

Reply:
Makes sense. And thanks for clarifying it doesn't enforce our customers to adopt this version.

I'm happy with the change, but in future, I would prefer doing it in a separate PR to make changes and release notes more transparent.